doc/Cookbook/Single_DiffIm_Recipe.md

```diff
@@ -0,0 +1,32 @@
+# Recipe: Image Subtraction between Two Single-sensor Visit Images
+
+This describes how to run the LSST Image Processing Stack program on two different Twinkles exposures. 
+
+## Set up the LSST environment
+The first step required is to set up the `lsst` stack on NERSC or a computer where the data can be accessed. For details on how to set up packages in the `lsst` stack, please go to [page](here). For what we need, we will have to set up both the DM stack and the Twinkles software:
```

Comment:
We already have an installation at NERSC (and SLAC).  This is really a very specific tutorial for people getting the pipelines up and running, so I think it's o.k. to forego general instructions about setting up the stack and use very specific instructions for completing these tasks with the system as deployed on the systems we are using.

In other words, let's remove the parts that talk about how to set up the LSST environment in general and replace them with specifics for our system, like what Michael put on his confluence page: https://confluence.slac.stanford.edu/display/LSSTDESC/Twinkles+Subtractions